app/_redirects

```diff
@@ -5,6 +5,11 @@
 # FREE TRIALS, GENERAL LICENSING
 /free-trial/guide/                    https://konghq.com/install
 /konnect-platform/plans/              https://konghq.com/pricing/
+/konnect-platform/get-started/        /konnect/getting-started/
+/konnect-platform/compatibility/plugins/  /hub/plugins/compatibility/
+/konnect-platform/key-concepts/           /konnect/availability-stages/
+/konnect-platform/support-policy/         /gateway/latest/support-policy/
```

Comment:
The URL redirected to the right page, the page just results in a 404.

Reply:
That's because I only just rebased this topic before our meeting and this change is currently in 2.8.x, not 3.0.x. I'll add it to 3.0.x as well and it should work.